sql/_13_issues/_23_1h/cases/cbrd_24419.sql

```diff
@@ -508,8 +508,8 @@ select * from db_auth where class_name in (select class_name from db_class where
 select * from db_index where class_name in (select class_name from db_class where is_system_class = 'NO') order by owner_name, class_name, index_name;
 select * from db_index_key where class_name in (select class_name from db_class where is_system_class = 'NO') order by owner_name, class_name, index_name;
 select * from db_trig order by owner_name, trigger_name;
-select * from db_stored_procedure order by sp_name;
-select * from db_stored_procedure_args order by sp_name;
+select sp_name, sp_type, return_type, arg_count, lang, target, owner, comment from db_stored_procedure order by sp_name;
+select sp_name, index_of, arg_name, data_type, mode, comment from db_stored_procedure_args order by sp_name;
```

Comment:
I think it would be better to modify it as follows.

select sp_name, index_of, arg_name, data_type, mode, comment from db_stored_procedure_args order by sp_name, index_of ;